### ISSUE TITLE
Some Glish words clash with existing monosyllable word

### ISSUE THREAD

**Issue by Aikoyori** [paralogical/glish#4]
For example, the word floor and kink exist as a real, monosyllable words. Revolution and Kinda are non-monosyllable words that got truncated to aforementioned two words.

## image
<img width="1085" alt="image" src="https://github.com/paralogical/glish/assets/12034280/74b7ede9-9462-4b40-96d3-3ea1674c2fe6">

**Comment by Windowsfreak** [comment]
While "kink" is a word that may or may not exist depending on who and where you ask, you're probably right with floor. I assume the IPA has a different idea of how the "floor" of revolution ought to be pronounced 🤷🏻‍♂️

**Comment by paralogical** [comment]
yeah, I think like https://github.com/paralogical/glish/issues/6, it's an issues with the IPA -> english-like spelling that makes it look the same. This would maybe go away if we respelled all the already monosyllabic words, but then it would be even harder to read.

**Comment by paralogical** [comment]
I think revolution is "intended" by glish to be read like "flew-er", but the original english "floor" is more like "flor"
